sql: add trace statement document (#890)
pingcap/docs: sql/util.md

## `TRACE` statement

```sql
TRACE [FORMAT = format_name] traceable_stmt

format_name:
    "json" | "row"

traceable_stmt: {
    SELECT statement
  | DELETE statement
  | INSERT statement
  | REPLACE statement
  | UPDATE statement
}
```

```sql
mysql> trace format = 'row' select * from mysql.user;
+---------------------------|-----------------|------------+
| operation                 | startTS         | duration   |
+---------------------------|-----------------|------------+
| session.getTxnFuture      | 19:54:35.310841 | 4.255µs    |
|   ├─session.Execute       | 19:54:35.310837 | 928.349µs  |
|   ├─session.ParseSQL      | 19:54:35.310906 | 35.379µs   |
|   ├─executor.Compile      | 19:54:35.310972 | 420.688µs  |
|   ├─session.runStmt       | 19:54:35.311427 | 222.431µs  |
|   ├─session.CommitTxn     | 19:54:35.311601 | 14.696µs   |
|   ├─recordSet.Next        | 19:54:35.311828 | 419.797µs  |
|   ├─tableReader.Next      | 19:54:35.311834 | 379.932µs  |
|   ├─recordSet.Next        | 19:54:35.312310 | 26.831µs   |
|   └─tableReader.Next      | 19:54:35.312314 | 2.84µs     |
+---------------------------|-----------------|------------+
10 rows in set (0.00 sec)
```

> When the `format` is `json`, the output is some `json` text. If the text is too long, they are split into multiple lines.

Reviewer: ```suggestion
When the `format` is `json`, the output is some `json` text. If the text is too long, it is split into multiple lines.
```

> The `json` output can be viewed in the Web UI which is integrated into TiDB. Here is a demo:

Reviewer 2: We should explain what the web UI is with consistent terminology.  For example:
```suggestion
A visualization of the `json` output can be accessed via the `tidb-server` status and metrics port (default: `:10080`). Here is a demo:
```

Author: Done

> The `json` output can be viewed in the Web UI which is integrated in TiDB, here is an demo:

Reviewer: ```suggestion
The `json` output can be viewed in the Web UI which is integrated into TiDB. Here is a demo:
```
